packages/training/python/src/training/service_manager.py

```
@@ -316,6 +326,10 @@ def _start_vllm(self) -> bool:
         logger.info(f"Starting vLLM server on port {port}...")
         logger.info(f"  Model: {cfg.model_name}")
         logger.info(f"  GPU Memory: {cfg.vllm_gpu_memory_utilization * 100:.0f}%")
+        if cfg.tensor_parallel_size > 1:
+            logger.info(f"  Tensor Parallel: {cfg.tensor_parallel_size} GPUs")
+        if cfg.use_flash_attention:
+            logger.info(f"  Flash Attention: enabled")
```

Ruff (0.14.10)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)